bindings/node/test/clientEncryption.test.js

```diff
@@ -763,6 +765,8 @@ describe('ClientEncryption', function () {
         queryType: 'rangePreview',
         contentionFactor: 0,
         rangeOptions: {
+          min: new BSON.Int32(0),
+          max: new BSON.Int32(10),
```

Comment:
I guess index.d.ts also needs an update?

Reply:
Tests just started failing with https://github.com/mongodb/libmongocrypt/commit/aa145adb538aeaf530e530ee801ef81391e655b4 so needed to update.

Reply:
The types indicate that `min` and `max` are optional, though – is that still accurate? (It's fine if it is, I just assumed that it isn't anymore based on the PR title)

Reply:
Ah... Good catch. They are required now.